custom_components/sector/const.py

```diff
@@ -31,3 +31,26 @@
 
 CONF_PANEL_ID = "panel_id"
 CONF_CODE_FORMAT = "code_format"
+
+BINARY_SENSOR_DESCRIPTIONS = (
```

Comment:
We should keep it in `binary_sensor.py`. Just constants should be before any of the functions starts.